src/SOS/SOS.UnitTests/Scripts/StackTests.script

```diff
@@ -66,6 +66,8 @@ VERIFY:\s+[r|e]ax=<HEXVAL>\s+[r|e]bx=<HEXVAL>\s+[r|e]cx=<HEXVAL>\s+
 ENDIF:64BIT
 ENDIF:PROJECTK
 
+IFDEF:NEVER
```

Comment:
What are these annotations? Hadn't seen them before

Reply:
I kinda cheated here. I using a undefined "define" which is never true.

Reply:
They will go away when I fix  the clrstack -i problems.

Reply:
https://github.com/dotnet/diagnostics/issues/266

Reply:
I added a comment to the scripts with the issue.